backend/src/db/migrations/20250414203701_add-notification-flag-service-token.ts

```diff
@@ -0,0 +1,27 @@
+import { Knex } from "knex";
+
+import { TableName } from "../schemas";
+
+export async function up(knex: Knex): Promise<void> {
+  const hasCol = await knex.schema.hasColumn(TableName.ServiceToken, "notificationSent");
```

Comment:
it helps to be more descriptive about column names... maybe we could call this `expiryNotificationSent` instead?